wallets/metamask/src/prepareExtension.ts

```diff
@@ -1,14 +1,25 @@
+import path from 'node:path'
 import { downloadFile, ensureCacheDirExists, unzipArchive } from '@synthetixio/synpress-cache'
+import fs from 'fs-extra'
 
 export const DEFAULT_METAMASK_VERSION = '11.9.1'
 export const EXTENSION_DOWNLOAD_URL = `https://github.com/MetaMask/metamask-extension/releases/download/v${DEFAULT_METAMASK_VERSION}/metamask-chrome-${DEFAULT_METAMASK_VERSION}.zip`
 
-export async function prepareExtension() {
-  const cacheDirPath = ensureCacheDirExists()
+export async function prepareExtension(forceCache = true) {
+  let outputDir = ''
+  if (forceCache) {
+    outputDir = ensureCacheDirExists()
+  } else {
+    outputDir = path.resolve('./', 'downloads')
```

Comment:
can we ensure we use platform agnostic path resolutions? 

Based on the patterns from #1175 we can now use Synpress directly on Windows, this wouldn't be compatible. Check the linked PR on how to do that.

Reply:
@matstyler worth fixing?
example:
https://github.com/Synthetixio/synpress/pull/1175/files#diff-771d1fb7e6025d048bb38f171dfa5fb00f97044447ec1e5c2495312e283a9a0aR13-R14